inst/extdata/gcam-usa/EIA_AEO_Tab4.csv

```diff
@@ -0,0 +1,4 @@
+# File: EIA_AEO_Tab4.csv
+# Title: AEO Table 4: Residential Energy Use 
+# Source: AEO 1996-2011
```

Comment:
Is there a link, DOI, ISBN, or something can add here?

Reply:
My understanding is that this is compiled from a bunch of different AEOs. Not sure if there is any good single place to point to.